apps/topicprio/templates/meinberlin_topicprio/includes/topic_mgmt_list_item.html

```diff
@@ -1,21 +1,20 @@
 {% load i18n %}
 
 <div class="list-item list-item--squashed">
-    <h3 class="list-item__title">
-        <a href="{% url 'meinberlin_topicprio:topic-mgmt-update' slug=object.slug %}">
-        {{ object.name }}
-        </a>
-    </h3>
-    {% if object.category %}
-        <span class="label label--big">{{ object.category }}</span>
-    {% endif %}
-    <div class="list-item__actions">
-        <span class="list-item__action">
+    <div class="lr-bar">
+        <div class="lr-bar__left">
+            <h3 class="list-item__title">
+                <a href="{% url 'meinberlin_topicprio:topic-mgmt-update' slug=object.slug %}">
+                {{ object.name }}
+                </a>
+            </h3>
+        </div>
+        <div class="lr-bar__right">
             <a href="{% url 'meinberlin_topicprio:topic-mgmt-delete' slug=object.slug %}"
                class="button button--light">
                 <i class="fa fa-remove" aria-hidden="true"></i>
                 {% trans 'Delete'%}
             </a>
-        </span>
+        </div>
```

Comment:
I also thought about this, but decided against it because `lr-bar__right` moves down on if `lr-bar__left` has more than one line. Of course we could leave this as it is and change the `lr-bar` implementation instead.